skyplane/api/impl/transfer_job.py

```diff
@@ -56,6 +56,41 @@ def estimate_cost(self):
         # TODO
         raise NotImplementedError
 
+    def calculate_size(self):
```

Comment:
We should cache this from the Chunker class and reuse it. list_objects is slow so we should only call it once. This may mean we only report usage after a transfer completes which is OK.